### PR TITLE
[CHI-308] Fixed modal scrolling and positioning issue

### PR DESCRIPTION
**Pull request by janibolkvadze**
@jllr @dani-cl-madrid @mattnickles @SergioQCL 

https://ctl.atlassian.net/browse/CHI-308

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/300/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/301/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/303/ for more details.

File: src/chi/components/modal/modal.scss [hunk 1]
```diff
@@ -7,6 +7,15 @@ $animation-duration: 0.5s;
   overflow: hidden;
 }
 
+.-long-content-aligner {
```
Comment (jllr) [inline src/chi/components/modal/modal.scss]: Doubting about this class name

File: src/chi/components/modal/modal.scss [hunk 2]
```diff
@@ -7,6 +7,15 @@ $animation-duration: 0.5s;
...
+  align-items: flex-start;
+  overflow-x: hidden;
+  overflow-y: auto;
+  & .a-modal {
+    margin-top: 1rem;
```
Comment (jllr) [inline src/chi/components/modal/modal.scss]: Please add margin-bottom: 1rem

File: src/chi/components/modal/modal.scss (hunk 1 above)
Comment (dani-cl-madrid) [reply src/chi/components/modal/modal.scss]: @jllr If -disableScroll is camel case, -long-content-aligner should be as well, or the opposite.

File: src/chi/javascript/components/modal.js [hunk 3]
```diff
@@ -110,9 +111,24 @@ class Modal extends Component {
...
+  getElementHeight(el) {
+    let height;
+    const clone = el.cloneNode(true);
+
+    document.body.append(clone);
+    height = parseInt(window.getComputedStyle(clone).getPropertyValue('height').replace("px", ""));
+    clone.remove();
+    
+    return height;
+  }
```
Comment (dani-cl-madrid) [inline src/chi/javascript/components/modal.js]: An element in the Dom, cloned at the end of the document body might have different height, as the width might be different. Please, use Util:calculateExternalHeight or any equivalent solution.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/310/ for more details.

**Comment by jllr** [comment]
@janibolkvadze please solve conflicts

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/317/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/318/ for more details.

**Pull request closed without merging**